### DISCUSSION TITLE
[Question]: alternative to Google Coral?

### DISCUSSION THREAD

**Discussion by gtramsey2000** [blakeblackshear/frigate#12962] [Ask A Question]
### What is your question:

I've been fighting with my machine all day to get the pcie hardware to work.  I've found countless people having the same issue and Google appears to have abandoned the tech.  Is there anything that people are using as a better, more up to date/supported alternative?

**Comment by NickM-27** [comment]
There are many options listed at https://docs.frigate.video/configuration/object_detectors

**Comment by jimtng** [comment]
I'm using an NVidia T600 card. Seems to work fine for me.

**Comment by ForTheRush** [comment]
The PCI coral is definitely a pain in the ass to get working, took me about a week but once it works, it's very much set it and forget it. I have both USB and PCI running and theyre damn near identical inference speeds. I pull 4-6ms inference speeds which is better than any other detector I've tried
